### PR TITLE
Add 10.6.6 Studio Pro release notes

### PR DESCRIPTION
**Pull request by MariaShaposhnikova**

### CODE REVIEW

File: content/en/docs/releasenotes/studio-pro/10/10.6.md [hunk 1]
```diff
@@ -15,6 +15,41 @@ weight: 94
...
+
+**Release date: March ___, 2024**
+
+{{% button color="info" href="https://marketplace.mendix.com/link/studiopro/10.6.6" text="Go to Marketplace" title="Download version 10.6.6 from the Marketplace" %}}
+
+### Improvements
+
+* We have removed react-native-code-push dependency. We recommend migrating to Mendix OTA instead.
+* We improved the memory consumption in Studio Pro by resolving a memory leak caused by a rebase notification bar.
+
```
Comment (arjenw) [inline content/en/docs/releasenotes/studio-pro/10/10.6.md:32]: @MariaShaposhnikova can you also add the following release note here?

```
#### WebSQL replacement

We introduced an alternative for WebSQL with this release. After upgrading to this release, opening your offline PWA in your browsers automatically upgrades your WebSQL database to a new SQLite-based database. For this to function correctly, the browser must run in a Secure Context, which means either an HTTPS connection or by accessing your offline PWA via localhost. All functionality will remain functioning as is.
```

Comment (MariaShaposhnikova) [reply content/en/docs/releasenotes/studio-pro/10/10.6.md:32]: @arjenw , we announced it in 10.9 as well, should it stay in only one of them or both?

Comment (arjenw) [reply content/en/docs/releasenotes/studio-pro/10/10.6.md:32]: In both as this is the MTS release in which this is added too.

Comment (MariaShaposhnikova) [reply content/en/docs/releasenotes/studio-pro/10/10.6.md:32]: Added.

**Comment by MariaShaposhnikova** [comment]
Needs a date update, the rest is good to go.

**Pull request merged**